package/gluon-mesh-vpn-wireguard/Makefile

```diff
@@ -0,0 +1,19 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-mesh-vpn-wireguard
+PKG_VERSION:=3
+
+include ../gluon.mk
+
+PKG_CONFIG_DEPENDS += $(GLUON_I18N_CONFIG)
+
+define Package/gluon-mesh-vpn-wireguard
+  TITLE:=WireGuard Mesh VPN Support
+  DEPENDS:=+gluon-core +ip +kmod-wireguard +wireguard-tools +kmod-udptunnel6 +kmod-ipt-hashlimit +micrond
```

Comment:
This should probably conflict with L2 mesh protocols.

Reply:
wireguard does not support L2 ^^

Reply:
hence the term "conflict".

Reply:
ok.